src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -87,18 +87,15 @@ export interface ContinuationInFilingIF {
     identifier: string
   }
   continuationIn: {
-    foreignJurisdiction: any
+    existingBusinessInfo: any // FUTURE: create an interface for this object
+    continuationAuthorization: any // FUTURE: create an interface for this object
```

Comment:
We need to figure out what else goes into each of these object, the schema, etc. But not in this ticket.